service/controller/v5/resource/instance/types.go

```diff
@@ -38,5 +40,8 @@ func newNodeOSImageCoreOS() nodeOSImage {
 // SmallCloudconfigConfig represents the data structure required for executing
 // the small cloudconfig template.
 type SmallCloudconfigConfig struct {
-	BlobURL string
+	BlobURL       string
```

Comment:
For me again, why is the blob URL necessary here?

Reply:
There is a limit for the size of the machine scaleset userdata field. We can't put full ignition there. So we put small ignition, which then appends main ignition config from url. This small ignition also includes an encryption key for certificates.

Reply:
It is the same approach as in aws-operator https://github.com/giantswarm/aws-operator/blob/master/service/controller/v23/templates/cloudconfig/small.go#L9